README.md

```diff
@@ -354,7 +356,80 @@ The `extract` method takes no arguments:
     1. `v` must have the same type that `f` returns in `extend`.
 
 
+### Profunctor
 
+A value that implements the Profunctor specification must also implement
+the Functor specification. The Profunctor specification is a Contravariant
+functor on its first type parameter and functor on its second type parameter.
+
+1. `p.dimap(f, g)` is equivalent to `p` (identity)
+2. `p.dimap(compose(f1)(f2)), compose(g1)(g2))` is equivalent to `p.dimap(f1, g1).dimap(f2, g2)` (composition)
```

Comment:
Also in the rest of the spec function composition is just 'hardcoded', instead of using a helper `compose` function.